list/README.md

```
@@ -600,6 +600,7 @@ Initially created by [Marko](https://markodenic.com) at [Web Development Resourc
 | [ToolsChimp.com](https://toolschimp.com/)                                        | 1000+ Free Online tools. No Signup, No Paywall. Color Converters, Image Tools, Video Tools, etc.,                         |
 | [volumeshaderbm.org](https://volumeshaderbm.org)                                 | A browser-based volume shader benchmark with interactive kernels and reproducible presets.                                |
 | [Format JSON Online](https://formatjsononline.com)                                 | A Complete Toolkit for JSON & Data Processing.|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)